Resources/Prototypes/_RMC14/Entities/Structures/Walls/walls.yml

```
@@ -90,7 +90,9 @@
 
 - type: entity
   parent: CMBaseWall
+  name: super reinforced wall
```

From 13 the name is just reinforced wall apparently
```suggestion
  name: reinforced wall
```